### PR TITLE
James/hardhat plugin

### PR DESCRIPTION
**Pull request by jim4067**

### CODE REVIEW

File: packages/hardhat-astar-plugin/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: https://EditorConfig.org
+root = true
+
+[*]
```
Comment (drewstone) [inline packages/hardhat-astar-plugin/.editorconfig]: You can remove this file and add to `.gitignore`

File: packages/hardhat-astar-plugin/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3471 @@
+{
```
Comment (drewstone) [inline packages/hardhat-astar-plugin/package-lock.json]: This file shouldn't be necessary afaik. Best to use yarn but overall should be contained in a master `yarn.lock` if the resolutions are properly set up.

File: packages/hardhat-astar-plugin/test/project.test.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        this.hre.config.paths.newPath,
+        path.join(process.cwd(), "asd")
+      );
+    });
+  });
+});
+
+describe("Unit tests examples", function () {
+  describe("ExampleHardhatRuntimeEnvironmentField", function () {
+    describe("sayHello", function () {
```
Comment (drewstone) [inline packages/hardhat-astar-plugin/test/project.test.ts]: These should be changed to use `sdk-core` API and test the evm conversion tools.

**Review by drewstone — CHANGES_REQUESTED** [review]

**Comment by drewstone** [comment]
CI has errors on tests and linting.

**Comment by jim4067** [comment]
> CI has errors on tests and linting.

Trying to fix them.

**Pull request merged**